### ISSUE TITLE
Failed to load Tensorflow Model IllegalArgumentException: Expected URL scheme 'http' or 'https' but no colon was found

### ISSUE THREAD

**Issue by tomerh2001** [mrousavy/react-native-fast-tflite#51]
I have a deployment very soon and I found out my frame processing isn't working in my release versions but it does work in my debug versions.

I checked the catlog of my app and I saw `Failed to load Tensorflow Model IllegalArgumentException: Expected URL scheme 'http' or 'https' but no colon was found`, I need to fix this ASAP, can you suggest a solution or even just a patch I could work with? Would very appreciate any help with this

```
2024-02-13 19:53:22.365 31683-31762 FrameProce...inRegistry com.x                    I  Looking up Frame Processor Plugin "scanOCR"...
2024-02-13 19:53:22.365 31683-31762 FrameProce...inRegistry com.x                    I  Frame Processor Plugin "scanOCR" found! Initializing...
2024-02-13 19:53:22.370 31683-31762 ReactNativeJS           com.x                    I  Loading Tensorflow Lite Model 19
2024-02-13 19:53:22.370 31683-31762 ReactNativeJS           com.x                    I  Resolved Model path: app_assets_documentdetection
2024-02-13 19:53:22.374 31683-31762 ReactNativeJS           com.x                    E  'Failed to load Tensorflow Model 19!', [Error: java.lang.IllegalArgumentException: Expected URL scheme 'http' or 'https' but no colon was found]
2024-02-13 19:53:22.381 31683-31683 PreviewView             com.x                    I  Creating PreviewView...
2024-02-13 19:53:22.382 31683-31683 PreviewView             com.x                    I  PreviewView is 0x0, rendering 1920x1080 content. Resizing to: 0x0 (COVER)
2024-02-13 19:53:22.383 31683-31683 CameraView              com.x                    I  Updating CameraSession...
2024-02-13 19:53:22.384 31683-31759 CameraSession           com.x                    I  configure { ... }: Waiting for lock...
2024-02-13 19:53:22.385 31683-31759 CameraSession           com.x                    I  configure { ... }: Updating CameraSession Configuration... Difference(deviceChanged=true, outputsChanged=true, sidePropsChanged=true, isActiveChanged=true)
2024-02-13 19:53:22.385 31683-31759 CameraSession           com.x                    I  Configuring inputs for CameraSession...
2024-02-13 19:53:22.385 31683-31759 Persistent...ureSession com.x                    D  --> setInput(0)
2024-02-13 19:53:22.385 31683-31759 CameraSession           com.x                    I  Destroying previous outputs...
2024-02-13 19:53:22.386 31683-31683 PreviewView             com.x                    I  Setting PreviewView ResizeMode to COVER...
2024-02-13 19:53:22.389 31683-31759 CameraSession           com.x                    I  Creating outputs for Camera #0...
2024-02-13 19:53:22.392 31683-31763 unknown:ReactNative     com.x                    E  console.error: Failed to load Tensorflow Model 19! Error: java.lang.IllegalArgumentException: Expected URL scheme 'http' or 'https' but no colon was found, js engine: hermes, stack:
                                                                                                    _construct@1:189687
                                                                                                    Wrapper@1:189333
                                                                                                    _callSuper@1:187344
                                                                                                    SyntheticError@1:188814
                                                                                                    reactConsoleErrorHandler@1:188479
                                                                                                    ?anon_0_@1:2159260
                                                                                                    asyncGeneratorStep@1:543122
                                                                                                    _throw@1:543424
                                                                                                    tryCallOne@53:15
                                                                                                    anonymous@139:26
                                                                                                    anonymous@1:206378
                                                                                                    _callTimer@1:205329
                                                                                                    _callReactNativeMicrotasksPass@1:205473
                                                                                                    callReactNativeMicrotasks@1:207457
                                                                                                    __callReactNativeMicrotasks@1:93414
                                                                                                    anonymous@1:92498
                                                                                                    __guard@1:93252
                                                                                                    flushedQueue@1:92409
```

**Comment by mrousavy** [comment]
Fixed in [1.2.0](https://github.com/mrousavy/react-native-fast-tflite/releases/tag/v1.2.0) (this PR: https://github.com/mrousavy/react-native-fast-tflite/pull/30). 💪 

If you appreciate my libraries and my quick fixes and help, please consider [💖 sponsoring me on GitHub 💖](https://github.com/sponsors/mrousavy) so I can keep doing what I'm doing!

**Comment by tomerh2001** [comment]
> Fixed in [1.2.0](https://github.com/mrousavy/react-native-fast-tflite/releases/tag/v1.2.0) (this PR: #30). 💪
> 
> If you appreciate my libraries and my quick fixes and help, please consider [💖 sponsoring me on GitHub 💖](https://github.com/sponsors/mrousavy) so I can keep doing what I'm doing!

Appreciate it man 🙏 I donated 4$
